tutorials/deutsch_jozsa/deutsch_jozsa.ipynb

```diff
@@ -0,0 +1,393 @@
+{
```

Comment:
<p>The Hadamard transform is unitary, but it is not Hermitian!</p><p> </p><p>Maybe link for the relevant cell instead of the note: "(Read Mathematical Description below to learn more)"</p><p>Check this:</p><p><a href="https://stackoverflow.com/questions/16630969/ipython-notebook-anchor-link-to-refer-a-cell-directly-from-outside" target="_blank">https://stackoverflow.com/questions/16630969/ipython-notebook-anchor-link-to-refer-a-cell-directly-from-outside</a></p><p> </p><p>I think it's a nice standard we can use in our materials</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/179/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
In this context, I remember Eden mentioning it was hermitian but it makes sense that it's unitary. Will make that change. Thanks for the point about the linking of cell!